src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package service.impl;
+
+import db.Storage;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.ReportService;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+import strategy.OperationStrategyImpl;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String COLUMNS_TITLE = "fruit,quantity";
+    private OperationStrategy operationStrategy = new OperationStrategyImpl();
+
+    @Override
+    public HashMap<String, Integer> createReportToStoreDB() {
```

Comment:
ReportServiceImpl should be responsible only for creating the report

Create interface like FruitShopService and move this method there
and there you can rewrite method like:
```
    @Override
    public void processOfOperations(List<FruitTransaction> fruitTransactions) {
        for (FruitTransaction fruitTransaction : fruitTransactions) {
            operationStrategy.get(fruitTransaction.getOperation()).handle(fruitTransaction);
        }
    }
```